components/Storyblok/SbCampaignPage/SbCampaignPage.tsx

```diff
@@ -1,29 +1,107 @@
-import React from 'react';
-import { storyblokEditable, type SbBlokData } from '@storyblok/react/rsc';
+import { type SbBlokData } from '@storyblok/react/rsc';
+import { PageLayout } from '@/components/Storyblok/partials/PageLayout';
+import { CampaignHero } from './CampaignHero';
+import { type HeroContentAlignmentType, type LogoAlignmentType } from './SbCampaignPage.styles';
 import { CreateBloks } from '@/components/CreateBloks';
-import { CampaignHeader, type CampaignHeaderProps } from '@/components/Storyblok/SbCampaignPage/CampaignHeader';
-import { IconCardSection, type IconCardSectionProps } from '@/components/Storyblok/partials/IconCardSection';
-import { Footer, type FooterProps } from '@/components/Storyblok/partials/Footer';
+import { type VisibleHorizontalType } from '@/components/Image';
+import { type ModTypeSizeTypes, type AllCardBgColorType } from '@/utilities/datasource';
+import { type SbImageType } from '../Storyblok.types';
 
-export type SbCampaignPageProps = CampaignHeaderProps & IconCardSectionProps & FooterProps & {
+/**
+ * The heroContentColor and barAlignment options are deprecated.
+ * We will now use heroBgColor to determine the text color, and heroContentAlignment for the alignment of the content box.
+ */
```

Comment:
I deprecated 2 storyblok option. `heroContentColor` is either black or white text, we can easily determine text color from the `heroBgColor` option automatically. As for the `barAlignment`, we can determine that using `heroContentAlignment` as well.